src/store/global/global.thunks.ts

```diff
@@ -10,21 +10,17 @@ import { push } from "connected-react-router";
 import { ethers } from "ethers";
 import { Block, TransactionReceipt, TransactionResponse } from "@ethersproject/providers";
 import Connector from "@walletconnect/web3-provider";
-
 import HermezABI from "@hermeznetwork/hermezjs/src/abis/HermezABI";
 import { TxType, TxState } from "@hermeznetwork/hermezjs/src/enums";
-
```

Comment:
We do keep the separation between external imports and internal ones

Reply:
Wait, then does this imply that we keep a certain order for imports? This is getting complicated...

Reply:
Basically it's

- External Imports (from npm dependencies)
- New Line
- Internal Imports

Reply:
Ok. Done!